CONTRIBUTING.md

```diff
@@ -1,62 +1,152 @@
-# Contributing
+# Contributing To Simple Icons
 
-This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidance below.
 
-  - Include the SVG namespace declaration `xmlns="http://www.w3.org/2000/svg"`. This is not required if the SVG is included inline on HTML5 web pages.
-  - Include a descriptive `<title>` element for accessibility. The `<title>` must have an ID of the format `simpleicon-<iconname>-title` - for example, `<title id="simpleicons-github-title">GitHub icon</title>`.
-  - Add `aria-labelledby="simpleicon-<iconname>-title"` and `role="img"` to the `<svg>` element to improve screen reader support.
-  - Use a `viewBox` of 24 by 24, following [Google’s Material Design guidelines for system icons](https://material.io/guidelines/style/icons.html#icons-system-icons).
-  - Ensure the icon is scaled to fit the viewbox, while preserving the icon's original proportions. This means the icon should be touching _at least_ two sides of the viewbox.
-  - Ensure the icon is horizontally and vertically centered.
-  - Ensure that all paths and strokes have been converted to fills.
-  - Ensure your SVG is monochromatic. Remove all fill colors so that icons default to black.
-  - Minify your SVG after exporting it. We recommend using [SVGO](https://github.com/svg/svgo) or [SVGOMG](https://jakearchibald.github.io/svgomg/) to automate this process. **Do not go below a precision of 3**.
+## Table of contents
 
-## JSON data for simpleicons.org
+* [How To Request An Icon](#how-to-request-an-icon)
+* [How To Add An Icon](#how-to-add-an-icon)
+* [How To Label Issues](#how-to-label-issues)
+* [How To Build Locally](#how-to-build-locally)
 
-In addition to following the guidelines for SVGs, list new icons in the `_data/simple-icons.json` file. Each icon in the array has three required values:
+## How To Request An Icon
 
-  - The `title` of the new SVG.
-  - A `hex` color value that matches the brand or logo's main accent color. (Without the `#` pound symbol.)
-  - The `source` URL of the logo being used.
+We welcome icon requests. However, before you submit a new issue please make sure the icon:
 
-Here is the object for the GitHub logo as an example:
+* Is a _popular_ brand.
+* Has not already been requested.
+* Can be represented in a monochromatic format.
 
-```json
-{
-  "title": "GitHub",
-  "hex": "181717",
-  "source": "https://github.com/logos"
-},
-```
+When submitting a new icon request, please submit the issue with the following information:
+
+* **Title:** Include the brand name. Example: `Request: GitHub Icon`
+* **Body:** Include links to official sources for the brand's icon and official colors (e.g. media kits, brand guidelines, SVG files etc.)
+
+## How To Add An Icon
```

Comment:
Maybe change this to `How to add or change an icon`?

Reply:
Tweaked.